pandas/tests/groupby/test_groupby.py

```diff
@@ -873,11 +868,21 @@ def test_omit_nuisance(df):
         grouped.agg(lambda x: x.sum(0, numeric_only=False))
 
 
-def test_omit_nuisance_sem(df):
-    # GH 38774 - sem should work with nuisance columns
+@pytest.mark.parametrize(
```

Comment:
can you use this fixture: all_numeric_reductions

Reply:
@jreback do you mean I should add a fixture with that name, all_numeric_reductions? I cannot find that fixture inside the groupby folder

Reply:
its in the top level

Reply:
I found it. It is defined as 
`_all_numeric_reductions = [
    "sum",
    "max",
    "min",
    "mean",
    "prod",
    "std",
    "var",
    "median",
    "kurt",
    "skew",
]`

However, max and min do not drop column B which has dtype object. Also 'DataFrameGroupBy' object has no attribute 'kurt'

Reply:
can you add *another* test and parameterize on min/max that they *don't* drop (and put a comment that links to this issue)